indexer/src/api/v1/self_stash.rs

```diff
@@ -106,19 +106,19 @@ async fn get_self_stash_by_owner(
     };
 
     // Decode scope hex if provided (max 510 hex chars = 255 bytes)
-    if params.scope.clone().unwrap_or_default().len() > 255 {
+    if params.scope.clone().len() > 255 {
         return Err((
             StatusCode::BAD_REQUEST,
             Json(ErrorResponse {
-                error: "Alias hex length cannot exceed 32 characters".to_string(),
+                error: "Scope hex length cannot exceed 32 characters".to_string(),
             }),
         ));
     }
 
     let mut scope_bytes = [0u8; 255];
     match faster_hex::hex_decode(
-        params.scope.clone().unwrap_or_default().as_bytes(),
-        &mut scope_bytes[..params.scope.unwrap_or_default().len() / 2],
+        params.scope.clone().as_bytes(),
```

Comment:
no need to clone here

Reply:
fixed in [4eb236f62fd863a7a5bf8fab190e63290ab740f2](https://github.com/K-Kluster/kasia-indexer/pull/10/commits/4eb236f62fd863a7a5bf8fab190e63290ab740f2)